plugins/Jobs.cpp

```diff
@@ -239,7 +239,7 @@ bool BedrockPlugin_Jobs::peekCommand(SQLite& db, BedrockCommand& command) {
                 }
 
                 // If there's no job or the existing job doesn't match the data we've been passed, escalate to master.
-                if (!result.empty() && result[0][1] == job["data"]) {
+                if (!result.empty() && result[0][1] == (job["data"].empty() ? SToStr("{}") : job["data"])) {
```

Comment:
```plugins/Jobs.cpp:241:73: error: operands to ?: have different types ‘const char [3]’ and ‘std::map<std::basic_string<char>, SString, STableComp>::mapped_type {aka SString}’``` 

The actual compile error is a bit more telling. With ternary functions you can't have one side be a char* and the other side be a std::string (or SString or whatever). To fix that (without converting it) you can break it up into two `&&`s (which you might want to do anyway, not in love with a ternary inside an if).